Xamarin.MacDev/PListObject.cs

```diff
@@ -2463,15 +2444,15 @@ internal PObjectContainerEventArgs (PObjectContainerAction action, string key, P
 			get; private set;
 		}
 
-		public string Key {
+		public string? Key {
```

Comment:
Could we just ignite this to string.Empty  and remove the ?

Reply:
No:

`PObjectContainerEventArgs` is created from:

https://github.com/rolfbjarne/Xamarin.MacDev/blob/1fe7c16cd83bea35cc485c5a01999b78a5433fce/Xamarin.MacDev/PListObject.cs#L460

which is called with null values here:

https://github.com/rolfbjarne/Xamarin.MacDev/blob/1fe7c16cd83bea35cc485c5a01999b78a5433fce/Xamarin.MacDev/PListObject.cs#L450